metadata/metadata_test.go

```diff
@@ -26,6 +26,66 @@ import (
 
 var _ = Describe("Metadata", func() {
 
+	When("HasAnnotation/ HasAnnotationWithValue is called", func() {
+		It("should return true when the annotation with/ without the given value is found", func() {
+			pod := &corev1.Pod{
+				ObjectMeta: v1.ObjectMeta{
+					Annotations: map[string]string{"bar": "foo", "baz": "qux"},
+				},
+			}
+
+			found := HasAnnotation(pod, "bar")
+			Expect(found).To(BeTrue())
```

Comment:
Replace with `Expect(HasAnnotation(pod, "bar").To(BeTrue())`

Reply:
Updated, thanks!